vulcan/_client.py

```diff
@@ -18,8 +18,8 @@ class Vulcan:
     :var `~vulcan._data.VulcanData` ~.data: the data client
     """
 
-    def __init__(self, keystore, account, logging_level: int = None):
-        self._api = Api(keystore, account)
+    def __init__(self, keystore, account, session=None, logging_level: int = None):
```

Comment:
Move it after `logging_level`, otherwise it's incompatible for devs using it like `Vulcan(keystore, account, DEBUG)` etc.

Reply:
when devs upgrades lib version they should first review changelog for such changes, imo it's better to keep loggining level at the end and add breaking changes section to release notes instead of moving session